packages/jaeger-ui-components/tsconfig.json

```diff
@@ -1,4 +1,13 @@
 {
+  "compilerOptions": {
+    "baseUrl": "node_modules/@types",
+    "typeRoots": ["node_modules/@types"]
+  },
   "extends": "@grafana/tsconfig",
-  "include": ["src/**/*.ts*", "typings"]
+  "include": [
+    "src/**/*.ts*",
+    "typings",
+    "../../public/app/types/jquery/*.ts",
+    "../../public/app/types/sanitize-url.d.ts"
```

Comment:
why do we need these types here?

Reply:
The jquery type is needed here because of grafana/data. if we do not add it here it will cause type check error. I'm going to remove the other one.